v3/integration/config.json

```diff
@@ -426,6 +426,9 @@
       "ErrCount": 2
     },
     "e_ev_country_name_missing": {},
+    "e_ev_invalid_business_category": {
+      "ErrCount": 10957
```

Comment:
This lint seems fine, but 10k+ failed certificates is a bit smelly (unless this is a requirement that you happen to know is commonly failed).

Off the top of my head, either these 10k truly are failures or they are doing something like using mixed casing for these strings (which may-or-may-not be valid, but it would be good to know).

Would you mind running the integration tests locally? My typical workflow is to simply take a random sampling of certs that failed and inspect them closely.

```bash
cd v3/
make integration INT_FLAGS="-lintSummary -fingerprintSummary -lintFilter='e_ev_invalid_business_category'"
```

This will give you a printout of the fingerprints of the certificates that failed. You can then hop on over to a tool such as [crt.sh](https://crt.sh/) and plug that fingerprint in to see a copy of the offending cert as well as run linting from other similar tools to see what their opinion is.

Reply:
I understand your perplexity. I was a little too laconic in proposing this lint. As I said, I took inspiration from a specific incident reported on Bugzilla (bug #1894560). As you can see there, at least two root programs (Mozilla and Chrome) have made it clear that, like it or not, businessCategory must be case-sensitive. Yes, there are numerous mis-issued EV certificates in the test corpus. Probably, until two months ago several CAs had a different idea on the matter and issued EV certificates with businessCategory in uppercase, honestly thinking that it was okay; but unfortunately, in light of bug #1894560, one must conclude that this is not the case. Hence this lint. 

We must also consider that this lint detects not only the businessCategory values ​​that do not have the right casing, but also those that are completely wrong such as the following very weird case: 
https://crt.sh/?q=fffc65f342a9cc71bed437fea03845ddfa94f403edf1bc3182c0ae4baa0ff6d8 
(there are many instances like this one). 

Furthermore, this lint detects errors that are almost invisibile at first sight, such as when the dash in "Non-Commercial Entity" is not the normal minus sign character (0x2D) but the Unicode HYPHEN character (0xE28090 in UTF-8). It seems we have a number of this cases in the testCorpus, unfortunately. Other linters, such as Pkilint, also flag it as an error.

Reply:
Thank you for the rich context and history!

I figured that with so many errors that it had to be something as inane as casing or fiddly-bits such as extra whitespace or misunderstood character encodings. Plain text encoding for machine consumption truly is the devil.